tests/JustSaying.IntegrationTests/Fluent/MessagingBusBuilderTests.cs

```diff
@@ -119,15 +119,18 @@ public async Task Can_Create_Messaging_Bus_Fluently_For_A_Topic_Address()
         var topicName = Guid.NewGuid().ToString();
 
         // Arrange
+        var bus = new InMemoryAwsBus();
         var services = new ServiceCollection()
             .AddLogging((p) => p.AddXUnit(OutputHelper))
             .AddJustSaying(
                 (builder) =>
                 {
                     builder
                         .Client((options) =>
-                            options.WithBasicCredentials("accessKey", "secretKey")
-                                .WithServiceUri(TestEnvironment.SimulatorUrl))
+                            options.WithClientFactory(() => new LocalAwsClientFactory(bus)))
+                            // TODO Add back LocalStack config for running in CI
+                            // options.WithBasicCredentials("accessKey", "secretKey")
+                            //     .WithServiceUri(TestEnvironment.SimulatorUrl))
```

Comment:
This might be better as a GH issue rather than/as well as a comment if we want to add back LocalStack in CI